content/en/user-guide/lambda-tools/hot-reloading/index.md

```diff
@@ -310,7 +303,7 @@ awslocal lambda create-function \
     --function-name hello-world \
     --runtime "nodejs16.x" \
     --role arn:aws:iam::123456789012:role/lambda-ex \
-    --code S3Bucket="hot-reload",S3Key="$(PWD)/dist" \
```

Comment:
The `$(PWD)` works pretty well with Linux/macOS. Can we have multiple tabs for different OS?

Reply:
`PWD` command does not exist under Ubuntu.

![image](https://github.com/localstack/docs/assets/97876429/ea3fd1cc-522e-4af1-a75c-6adffa724822)